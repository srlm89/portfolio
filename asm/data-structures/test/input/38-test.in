ERASE actual/38-test.out
INSERT 13 5.083055 Bangkok Dangerous 720p
PRINT_IN_ORDER actual/38-test.out
INSERT 32 5.973325 Terminator 3 Rise Of The Machine [2003]   720p
INSERT 6 6.767690 Corpse Bride   720p
REMOVE_ID 32
PRINT_IN_ORDER actual/38-test.out
INSERT 5 2.847938 Armageddon [1998] [720p]
INSERT 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p
INSERT 11 5.776801 Godzilla 720p
REMOVE_ID 13
PRINT_IN_ORDER actual/38-test.out
INSERT 37 0.821835 Rambo 4 [2008] [720p]
INSERT 43 5.331704 Big Nothing   720p
INSERT 10 1.822436 Madagascar 1 [2005] [1080p]
INSERT 18 4.975778 Spiderman 2   1080p
REMOVE_ID 18
REMOVE_ID 37
PRINT_IN_ORDER actual/38-test.out
INSERT 22 9.455184 Mr. & Mrs. Smith   720p
INSERT 41 3.620573 Iron Man    1080p
INSERT 29 1.987359 Con Air   720p
INSERT 33 9.242183 Alien At The Attic 720P
INSERT 47 6.686024 Bolt [2008] [1080p]
REMOVE_ID 22
REMOVE_ID 11
PRINT_IN_ORDER actual/38-test.out
INSERT 4 9.102285 James Bond 18 - Tommorow Never Dies [1997] [720p]
INSERT 50 5.223409 Sex drive   720p
INSERT 1 6.803249 Miami Vice 720p
INSERT 21 0.297904 Fearless - 720p
INSERT 49 9.254606 Yes Man   720p
INSERT 39 3.263025 X-Men 4 - Origins Wolverine (2009)  1080p
REMOVE_ID 10
REMOVE_ID 50
REMOVE_ID 21
PRINT_IN_ORDER actual/38-test.out
INSERT 54 0.376743 Die Hard 4   1080p
INSERT 35 5.253051 The Notebook [2004] [1080p]
INSERT 15 9.027597 Horton Hears a Who! [2008] [720p]
INSERT 0 9.348749 The Mist   720p
INSERT 52 6.312547 Butterfly on a Wheel.2007   720p
INSERT 55 8.845485 Fast and the Furious 4 - 1080p
INSERT 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR
REMOVE_ID 6
REMOVE_ID 43
REMOVE_ID 39
PRINT_IN_ORDER actual/38-test.out
INSERT 23 1.783336 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 46 6.123678 Before The Devil Knows, You're Dead   1080p
INSERT 25 0.916872 Street Fighter The Legend Of Chun Li 720P
INSERT 44 4.106411 Rambo 4 [2008] [720p]
INSERT 56 0.647007 Next 720P
INSERT 17 0.983760 Dragonball Evolution [2009] [720p]
INSERT 26 1.906405 Disturbia    720p
INSERT 19 9.963789 Aladdin [1992] [720p]
REMOVE_ID 23
REMOVE_ID 33
REMOVE_ID 25
REMOVE_ID 29
PRINT_IN_ORDER actual/38-test.out
INSERT 70 1.115759 Black Hawk Down   1080p
INSERT 45 5.115561 New York I Love You 720p   720p
INSERT 16 1.327012 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN]
INSERT 68 1.851950 Gran Torino [2008] [720p]
INSERT 31 5.305491 Public Enemies [720p]
INSERT 27 0.225073 National Geographic - The Koran [720p] [Documenter]
INSERT 73 4.487863 War [720p]
INSERT 63 2.344528 Bridge To Terabithia 720P
INSERT 38 4.084491 American Virgin 720P
REMOVE_ID 54
REMOVE_ID 0
REMOVE_ID 4
REMOVE_ID 52
PRINT_IN_ORDER actual/38-test.out
INSERT 80 3.100442 Minority Report [720p]
INSERT 60 8.347025 James Bond 22 - Quantum of Solace [2008] [720p]
INSERT 12 5.327840 Drag Me To Hell [720p]
INSERT 14 3.550969 A Knight's Tale [2001] [720p]
INSERT 28 3.552695 007.1983.Octopussy.720p.hdtv.x264
INSERT 42 3.767727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD
INSERT 88 9.581665 Eagle Eye    1080p
INSERT 72 7.076818 Stardust   720p
INSERT 61 1.102213 911 Mysteries Demolition    720p
INSERT 48 9.283970 Labor Pain   720p
REMOVE_ID 9
REMOVE_ID 44
REMOVE_ID 88
REMOVE_ID 41
REMOVE_ID 12
PRINT_IN_ORDER actual/38-test.out
INSERT 93 5.021756 Yes Man   720p
INSERT 81 0.030731 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
INSERT 30 7.673966 Moulin Rouge   720p
INSERT 96 0.855281 The Bank Job [720p]
INSERT 74 2.910227 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 62 2.125745 Star Wars Saga  Collection 720p
INSERT 53 7.963383 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 84 0.491484 Ice Age 3 Dawn of the Dinosaurs [2009] [720p]
INSERT 77 9.972323 Green Lantern First Flight   720p
INSERT 99 1.251404 Bride Wars   720p
INSERT 67 1.676636 Orphan (2009)    1080p
REMOVE_ID 5
REMOVE_ID 68
REMOVE_ID 61
REMOVE_ID 81
REMOVE_ID 15
PRINT_IN_ORDER actual/38-test.out
INSERT 59 0.878448 Memento    1080p
INSERT 92 4.511640 The Italian Job [2003] [720p]
INSERT 79 1.568621 The Last Samurai [2004] [720p]
INSERT 104 0.113552 Meet The Parents    720p
INSERT 106 7.826191 Leaving Las Vegas  720p
INSERT 103 6.362386 Lawrence of Arabia   1080p
INSERT 8 1.574758 The Butterfly Effect 3 Revelations 720p
INSERT 110 4.100601 The Matrix Revolution   1080p
INSERT 94 8.047675 Banlieue 13 - Ultimatum [720p]
INSERT 66 7.717026 Fifty Dead Men Walking   1080p
INSERT 65 5.798753 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD
INSERT 89 3.811490 21   1080p
REMOVE_ID 46
REMOVE_ID 49
REMOVE_ID 17
REMOVE_ID 1
REMOVE_ID 63
REMOVE_ID 55
PRINT_IN_ORDER actual/38-test.out
