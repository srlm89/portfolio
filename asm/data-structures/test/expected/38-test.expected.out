[ 13 5.083055 Bangkok Dangerous 720p ]
[ 6 6.767690 Corpse Bride   720p ][ 13 5.083055 Bangkok Dangerous 720p ]
[ 5 2.847938 Armageddon [1998] [720p] ][ 6 6.767690 Corpse Bride   720p ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 11 5.776801 Godzilla 720p ]
[ 5 2.847938 Armageddon [1998] [720p] ][ 6 6.767690 Corpse Bride   720p ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 10 1.822436 Madagascar 1 [2005] [1080p] ][ 11 5.776801 Godzilla 720p ][ 43 5.331704 Big Nothing   720p ]
[ 5 2.847938 Armageddon [1998] [720p] ][ 6 6.767690 Corpse Bride   720p ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 10 1.822436 Madagascar 1 [2005] [1080p] ][ 29 1.987359 Con Air   720p ][ 33 9.242183 Alien At The Attic 720P ][ 41 3.620573 Iron Man    1080p ][ 43 5.331704 Big Nothing   720p ][ 47 6.686024 Bolt [2008] [1080p] ]
[ 1 6.803249 Miami Vice 720p ][ 4 9.102285 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 5 2.847938 Armageddon [1998] [720p] ][ 6 6.767690 Corpse Bride   720p ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 29 1.987359 Con Air   720p ][ 33 9.242183 Alien At The Attic 720P ][ 39 3.263025 X-Men 4 - Origins Wolverine (2009)  1080p ][ 41 3.620573 Iron Man    1080p ][ 43 5.331704 Big Nothing   720p ][ 47 6.686024 Bolt [2008] [1080p] ][ 49 9.254606 Yes Man   720p ]
[ 0 9.348749 The Mist   720p ][ 1 6.803249 Miami Vice 720p ][ 4 9.102285 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 5 2.847938 Armageddon [1998] [720p] ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 15 9.027597 Horton Hears a Who! [2008] [720p] ][ 29 1.987359 Con Air   720p ][ 33 9.242183 Alien At The Attic 720P ][ 35 5.253051 The Notebook [2004] [1080p] ][ 41 3.620573 Iron Man    1080p ][ 47 6.686024 Bolt [2008] [1080p] ][ 49 9.254606 Yes Man   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 52 6.312547 Butterfly on a Wheel.2007   720p ][ 54 0.376743 Die Hard 4   1080p ][ 55 8.845485 Fast and the Furious 4 - 1080p ]
[ 0 9.348749 The Mist   720p ][ 1 6.803249 Miami Vice 720p ][ 4 9.102285 James Bond 18 - Tommorow Never Dies [1997] [720p] ][ 5 2.847938 Armageddon [1998] [720p] ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 15 9.027597 Horton Hears a Who! [2008] [720p] ][ 17 0.983760 Dragonball Evolution [2009] [720p] ][ 19 9.963789 Aladdin [1992] [720p] ][ 26 1.906405 Disturbia    720p ][ 35 5.253051 The Notebook [2004] [1080p] ][ 41 3.620573 Iron Man    1080p ][ 44 4.106411 Rambo 4 [2008] [720p] ][ 46 6.123678 Before The Devil Knows, You're Dead   1080p ][ 47 6.686024 Bolt [2008] [1080p] ][ 49 9.254606 Yes Man   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 52 6.312547 Butterfly on a Wheel.2007   720p ][ 54 0.376743 Die Hard 4   1080p ][ 55 8.845485 Fast and the Furious 4 - 1080p ][ 56 0.647007 Next 720P ]
[ 1 6.803249 Miami Vice 720p ][ 5 2.847938 Armageddon [1998] [720p] ][ 9 3.377454 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 15 9.027597 Horton Hears a Who! [2008] [720p] ][ 16 1.327012 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 17 0.983760 Dragonball Evolution [2009] [720p] ][ 19 9.963789 Aladdin [1992] [720p] ][ 26 1.906405 Disturbia    720p ][ 27 0.225073 National Geographic - The Koran [720p] [Documenter] ][ 31 5.305491 Public Enemies [720p] ][ 35 5.253051 The Notebook [2004] [1080p] ][ 38 4.084491 American Virgin 720P ][ 41 3.620573 Iron Man    1080p ][ 44 4.106411 Rambo 4 [2008] [720p] ][ 45 5.115561 New York I Love You 720p   720p ][ 46 6.123678 Before The Devil Knows, You're Dead   1080p ][ 47 6.686024 Bolt [2008] [1080p] ][ 49 9.254606 Yes Man   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 55 8.845485 Fast and the Furious 4 - 1080p ][ 56 0.647007 Next 720P ][ 63 2.344528 Bridge To Terabithia 720P ][ 68 1.851950 Gran Torino [2008] [720p] ][ 70 1.115759 Black Hawk Down   1080p ][ 73 4.487863 War [720p] ]
[ 1 6.803249 Miami Vice 720p ][ 5 2.847938 Armageddon [1998] [720p] ][ 14 3.550969 A Knight's Tale [2001] [720p] ][ 15 9.027597 Horton Hears a Who! [2008] [720p] ][ 16 1.327012 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 17 0.983760 Dragonball Evolution [2009] [720p] ][ 19 9.963789 Aladdin [1992] [720p] ][ 26 1.906405 Disturbia    720p ][ 27 0.225073 National Geographic - The Koran [720p] [Documenter] ][ 28 3.552695 007.1983.Octopussy.720p.hdtv.x264 ][ 31 5.305491 Public Enemies [720p] ][ 35 5.253051 The Notebook [2004] [1080p] ][ 38 4.084491 American Virgin 720P ][ 42 3.767727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 45 5.115561 New York I Love You 720p   720p ][ 46 6.123678 Before The Devil Knows, You're Dead   1080p ][ 47 6.686024 Bolt [2008] [1080p] ][ 48 9.283970 Labor Pain   720p ][ 49 9.254606 Yes Man   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 55 8.845485 Fast and the Furious 4 - 1080p ][ 56 0.647007 Next 720P ][ 60 8.347025 James Bond 22 - Quantum of Solace [2008] [720p] ][ 61 1.102213 911 Mysteries Demolition    720p ][ 63 2.344528 Bridge To Terabithia 720P ][ 68 1.851950 Gran Torino [2008] [720p] ][ 70 1.115759 Black Hawk Down   1080p ][ 72 7.076818 Stardust   720p ][ 73 4.487863 War [720p] ][ 80 3.100442 Minority Report [720p] ]
[ 1 6.803249 Miami Vice 720p ][ 14 3.550969 A Knight's Tale [2001] [720p] ][ 16 1.327012 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 17 0.983760 Dragonball Evolution [2009] [720p] ][ 19 9.963789 Aladdin [1992] [720p] ][ 26 1.906405 Disturbia    720p ][ 27 0.225073 National Geographic - The Koran [720p] [Documenter] ][ 28 3.552695 007.1983.Octopussy.720p.hdtv.x264 ][ 30 7.673966 Moulin Rouge   720p ][ 31 5.305491 Public Enemies [720p] ][ 35 5.253051 The Notebook [2004] [1080p] ][ 38 4.084491 American Virgin 720P ][ 42 3.767727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 45 5.115561 New York I Love You 720p   720p ][ 46 6.123678 Before The Devil Knows, You're Dead   1080p ][ 47 6.686024 Bolt [2008] [1080p] ][ 48 9.283970 Labor Pain   720p ][ 49 9.254606 Yes Man   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 53 7.963383 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 55 8.845485 Fast and the Furious 4 - 1080p ][ 56 0.647007 Next 720P ][ 60 8.347025 James Bond 22 - Quantum of Solace [2008] [720p] ][ 62 2.125745 Star Wars Saga  Collection 720p ][ 63 2.344528 Bridge To Terabithia 720P ][ 67 1.676636 Orphan (2009)    1080p ][ 70 1.115759 Black Hawk Down   1080p ][ 72 7.076818 Stardust   720p ][ 73 4.487863 War [720p] ][ 74 2.910227 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 77 9.972323 Green Lantern First Flight   720p ][ 80 3.100442 Minority Report [720p] ][ 84 0.491484 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 93 5.021756 Yes Man   720p ][ 96 0.855281 The Bank Job [720p] ][ 99 1.251404 Bride Wars   720p ]
[ 8 1.574758 The Butterfly Effect 3 Revelations 720p ][ 14 3.550969 A Knight's Tale [2001] [720p] ][ 16 1.327012 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 19 9.963789 Aladdin [1992] [720p] ][ 26 1.906405 Disturbia    720p ][ 27 0.225073 National Geographic - The Koran [720p] [Documenter] ][ 28 3.552695 007.1983.Octopussy.720p.hdtv.x264 ][ 30 7.673966 Moulin Rouge   720p ][ 31 5.305491 Public Enemies [720p] ][ 35 5.253051 The Notebook [2004] [1080p] ][ 38 4.084491 American Virgin 720P ][ 42 3.767727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 45 5.115561 New York I Love You 720p   720p ][ 47 6.686024 Bolt [2008] [1080p] ][ 48 9.283970 Labor Pain   720p ][ 51 0.245478 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 53 7.963383 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 56 0.647007 Next 720P ][ 59 0.878448 Memento    1080p ][ 60 8.347025 James Bond 22 - Quantum of Solace [2008] [720p] ][ 62 2.125745 Star Wars Saga  Collection 720p ][ 65 5.798753 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 66 7.717026 Fifty Dead Men Walking   1080p ][ 67 1.676636 Orphan (2009)    1080p ][ 70 1.115759 Black Hawk Down   1080p ][ 72 7.076818 Stardust   720p ][ 73 4.487863 War [720p] ][ 74 2.910227 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 77 9.972323 Green Lantern First Flight   720p ][ 79 1.568621 The Last Samurai [2004] [720p] ][ 80 3.100442 Minority Report [720p] ][ 84 0.491484 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 89 3.811490 21   1080p ][ 92 4.511640 The Italian Job [2003] [720p] ][ 93 5.021756 Yes Man   720p ][ 94 8.047675 Banlieue 13 - Ultimatum [720p] ][ 96 0.855281 The Bank Job [720p] ][ 99 1.251404 Bride Wars   720p ][ 103 6.362386 Lawrence of Arabia   1080p ][ 104 0.113552 Meet The Parents    720p ][ 106 7.826191 Leaving Las Vegas  720p ][ 110 4.100601 The Matrix Revolution   1080p ]
